src/poc.c

```diff
@@ -134,9 +135,9 @@ int main(int argc, const char **argv) {
         printf("Reading at malicious_x = %p... ", (void*)malicious_x);
         readMemoryByte(malicious_x++, value, score);
         printf("%s: ", (score[0] >= 2*score[1] ? "Success" : "Unclear"));
-        printf("0x%02X=a%ca score=%d ", 
+        printf("0x%02X=%c score='%d' ", 
                 value[0], 
-                (value[0] > 31 && value[0] < 127 ? value[0] : "a?a"), 
+                (value[0] > 31 && value[0] < 127 ? value[0] : '?'), 
```

Comment:
yep this was the one in which the gcc error came from before, I changed it to 'a?a' but this is much more clear